site/package.json

```diff
@@ -20,19 +20,22 @@
     "sync": "node ./sync.js",
     "clean": "rimraf dist"
   },
+  "engines": {
+    "node": ">=v12"
+  },
   "license": "MIT",
   "dependencies": {
-    "@compositor/x0": "^6.0.5",
-    "@rebass/mdx": "^1.0.0-1",
+    "@compositor/x0": "^6.0.7",
```

Comment:
We should think about removing the current toolset as `@compositor/x0` hasn't been updated in a while